src/vcr_clj/core.clj

```diff
@@ -33,7 +35,7 @@
                     (if-not (and *recording?* (apply recordable? args*))
                       (apply orig-fn args*)
                       (let [res (binding [*recording?* false]
-                                  (return-transformer (apply orig-fn args*)))
+                                  (return-transformer (apply (middleware orig-fn) args*)))
```

Comment:
If you want to preview a response body in a web browser, I'm thinking you'll need the middleware to be applied *after* the `return-transformer` so that the body has already been made re-readable, does that sound right?

Reply:
having the middleware see the transformed return value should also make it more consistent with the way it's called during playback.